utils/resb/src/binary/writer.rs

```diff
@@ -0,0 +1,993 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Not looking terribly closely at the writer; just make sure it is behind a feature flag.